### PR TITLE
Feat/move detector

### PR DESCRIPTION
**Pull request by ptr96**
motion detected, 
no sending frames to API

### CODE REVIEW

File: motion-detector/detection_events.py
Comment (xewerx) [inline motion-detector/detection_events.py:1]: is this file needed for some future implementation?

File: motion-detector/input/input.mp4
Comment (xewerx) [inline motion-detector/input/input.mp4:1]: you can set a name of file which will point what animal is on video

File: motion-detector/detection_events.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import sys
+
+def motion_events():
```
Comment (xewerx) [inline motion-detector/detection_events.py:3]: btw. function name should have verb which describe what function is doing

File: motion-detector/input/input.mp4
Comment (ptr96) [reply motion-detector/input/input.mp4:1]: renamed

File: motion-detector/detection_events.py (hunk 1 above)
Comment (ptr96) [reply motion-detector/detection_events.py]: feature removed

**Pull request merged**